### ISSUE TITLE
findElementBy..IfExists does not use the provided timeout.

### ISSUE THREAD

**Issue by zbranzov** [NativeScript/nativescript-dev-appium#250]
**Describe the bug**
Using await this._driver.findElementByAccessibilityIdIfExists("animatedBtn", 5000); does not wait 5 sec for the element. It terminates right away and the api does not have timeout param https://github.com/admc/wd/blob/13ffe77a1a599dcff5dcb8242c003e14cfcab57d/lib/commands.js#L1218
We have to implement it in the wrapper by using the https://github.com/admc/wd/blob/13ffe77a1a599dcff5dcb8242c003e14cfcab57d/lib/commands.js#L1218
